apps/prairielearn/elements/pl-number-input/pl-number-input.mustache

```diff
@@ -108,8 +124,22 @@
   {{/a_tru}}
 </span>
 {{/correct}}
-{{#partial}}<span class="badge badge-warning"><i class="fa fa-circle-o" aria-hidden="true"></i> {{partial}}%</span>{{/partial}}
-{{#incorrect}}<span class="badge badge-danger"><i class="fa fa-times" aria-hidden="true"></i> 0%</span>{{/incorrect}}
+{{#partial}}
```

Comment:
It looks like the feedback is only in the partial / incorrect case? The template should be such that there can always be feedback displayed (see other comments about correct answer feedback).

Reply:
Will do.

Reply:
Fixed in the latest commits.